src/frontend/apps/desk/src/components/__tests__/Modal.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { ModalSize } from '@openfun/cunningham-react';
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+
+import { AppWrapper } from '@/tests/utils';
+
+import { Modal, usePreventFocusVisible } from '../Modal';
+
+const TestComponent = () => {
+  usePreventFocusVisible(['.test-element']);
+
+  return (
+    <div>
+      <div className="test-element">Test Element</div>
+    </div>
+  );
+};
+
+describe('usePreventFocusVisible hook', () => {
+  test('sets tabindex to -1 on the target elements', () => {
+    render(<TestComponent />);
+
+    const testElement = screen.getByText('Test Element');
+
+    expect(testElement).not.toHaveAttribute('tabindex', '-1');
```

Comment:
This hook is too add a tabindex, no?
It should be that so, no ?: 
```suggestion
    expect(testElement).toHaveAttribute('tabindex', '-1');
```

If this test is complicated with Jest, the next test is maybe enough to assert that your code is working.

Reply:
My bad! I could mock it.